demo-instructions.md

```diff
@@ -83,48 +83,55 @@ Connecting the DICOM service to the existing network requires the configuration
-
+    1. Go to the App registrations, add find the Azure API for DICOM. ![Add API Permissions](demo-environment-images/add-api-permission.png)
+    2. Select DICOM.ReadWrite permissions and click Add permissions. ![Select DICOM.ReadWrite](demo-environment-images/add-dicom-read-write-permission.png)
     > Make note of the **application ID** and **client secret** when registering the client application.
     > 
     > Also, note the authorization URL from the **Endpoints** pane on the Overview page.  You'll need the **OAuth 2.0 token endpoint (v1)**
 
 9. Configure the web service connection.  This enables the router to communicate with the DICOM service.  
     1. Select the **MicrosoftDICOM** zone to expand the zone details.
-    ![Select Microsoft DICOM](/demo-environment-images/microsoft-dicom-zone.png)
+    ![Select Microsoft DICOM](demo-environment-images/microsoft-dicom-zone.png)
     2. Select **Web Service Connections** and then **MicrosoftDICOMWeb**.
-    ![Web service connections](/demo-environment-images/web-service-connections.png)
-    3. In the **Endpoint** URL box, enter the **DICOM service URL**.  You can find the URL in the Azure portal by browsing to **Resource Groups `>` rsna_workshop_studentN `>` DICOM Service `>` Overview `>` Service URL**.  
-    
-        > The service URL will be in a format like, https://myws-mydicom.dicom.azurehealthcareapis.com
-
-        ![Endpoint URL and authentication](/demo-environment-images/url-auth-details.png)
-
-    4. In the **Username** box, enter in the **Application ID** from the Azure Active Directory app registration process.  This is the ID the router uses to authenticate to the DICOM service.  
-    5. In the **Password** box, enter in the **Client secret** from the app registration process.  This is the client secret that is used to generate authentication tokens.  
-    6. In the **Auth URL** box, enter in the **Auth URL** for the DICOM service. 
-
-        > The auth URL should be in a format like, https://login.microsoftonline.com/0a0a0a0a-20ac-4545-9999-6ef19807994e/oauth2/token
-
-    7. From the menu bar, select **Save** to save the web service connection.
-    8. Test the connection by selecting the **Test OAuth** button (you may need to scroll down).  
-
-        ![Test OAuth](/demo-environment-images/test-oauth.png)
+    ![Web service connections](demo-environment-images/web-service-connections.png)
+    3. The following fields must be updated: **Endpoint URL**, **Username**, **Password**, and **Auth URL**.
+
+        ![Endpoint URL and authentication](demo-environment-images/url-auth-details.png)
+
+    4. In the Qvera **Endpoint URL** box, enter the Azure DICOM **Service URL**. You can find the URL in the Azure portal by browsing to **Resource Groups `>` rsna_workshop_studentN `>` DICOM Service `>` Overview `>` Service URL**.
```

Comment:
The resource group name won't use the "rsna" prefix, instead, it's whatever the user selects in the deployment template.  

```suggestion
    4. In the Qvera **Endpoint URL** box, enter the Azure DICOM **Service URL**. You can find the URL in the Azure portal by browsing to **Resource Groups `>` _your resource group_ `>` DICOM Service `>` Overview `>` Service URL**.
```